lib/plugin_tozny.dart

```diff
@@ -11,4 +12,46 @@ class PluginTozny {
     final String version = await _channel.invokeMethod('getPlatformVersion');
     return version;
   }
+
+  static Future<Record> readRecord(String recordID, ClientCredentials creds) async {
```

Comment:
Flutter by design is object-oriented. I really don't like that all of these methods are static. I realize this is just trying to get code out the door but there should really be a or tozny class that is instantiated and does the rest of these methods not needing to pass in a client credential.

Reply:
Agreed and will refactor here, I let usability/design slip in the "auto port all the methods" mode